test/gleam/erlang/os_test.gleam

```diff
@@ -0,0 +1,15 @@
+import gleam/erlang/os
+
+pub fn cmd_test() {
+  // Not sure Windows has an 'echo' command; might be *nix only.
+  assert "Hello, World!\n" = os.cmd("echo \"Hello, World!\"")
+}
+
+pub fn kernel_test() {
```

Comment:
What does this test do?

Reply:
Nothing, presently :laughing: This is what I get for writing things at 03:00. It's purpose is essentially to run `os.kernel` and make sure that whatever we get back makes some variety of sense, at least for a few cases.